akka-remote-tests/src/multi-jvm/scala/akka/remote/MultiNodeRemotingSpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.remote
+
+import java.util.UUID
+
+import akka.remote.testkit.{ FlightRecordingSupport, MultiNodeConfig, MultiNodeSpec, STMultiNodeSpec }
+import akka.testkit.{ DefaultTimeout, ImplicitSender }
+import com.typesafe.config.ConfigFactory
+import org.scalatest.{ Outcome, Suite }
+
+object MultiNodeRemotingSpec {
+
+  def arteryFlightRecordingConf =
+    ConfigFactory.parseString(
+      s"""
+        akka.remote.artery.advanced.flight-recorder {
+          enabled=on
+          destination=target/flight-recorder-${UUID.randomUUID().toString}.afr
+        }
+      """)
+
+}
+
+abstract class MultiNodeRemotingSpec(config: MultiNodeConfig) extends MultiNodeSpec(config)
```

Comment:
Essentially all the specs were mixing in the same thing, this aligns with the scalatest recommendation to define a base class instead (http://www.scalatest.org/user_guide/defining_base_classes). I have some vague memory of this speeding up compilation a bit but didn't check here.